tests/test_connector.py

```diff
@@ -0,0 +1,66 @@
+from logging import getLogger
+from unittest import mock
+
+from pytest import fixture, fail
+
+import code42_connector
+from phantom.action_result import ActionResult
+import phantom.app as phantom
+import py42.sdk
+from requests import HTTPError
+
+
+logger = getLogger(name=__name__)
+
+
+NULL_VALUE = object()
+
+
+@fixture
+def py42_sdk(mocker):
```

Comment:
I tend to think it is a good idea to distinguish pytest fixtures that are mocks from ones that are actual class instances (such as `connector`) via prefixing the mocks with `mock_`.